### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MarinaT-N**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -7,4 +7,34 @@ describe('getHumanAge', () => {
...
+    expect(goals).toBeInstanceOf(Array);
+  }); /* normaly this test is not necessary because of next test */
+
+  test('should return an expected HumanAge', () => {
+    const goals = getHumanAge(15, 15);
+
+    expect(goals).toEqual([1, 1]);
+  });
+
+  test('shoul return [0, 0], if cat&dog have 0 <= year < 1', () => {
```
Comment (Oleksandra-Pasenchenko) [inline src/getHumanAge.test.js:23]:
```suggestion
  test('should return [0, 0], if cat&dog have 0 <= year < 1', () => {
```

**Review by Oleksandra-Pasenchenko — CHANGES_REQUESTED** [review]
Please, focus on boundary values here, you should check all these requirements: 

Cat years are converted to human years following the next rules:
- First 15 cat years give 1 human year
- the next 9 cat years give 1 more human year
- every 4 next cat years give 1 extra human year

Dog years:
- First 15 dog years give 1 human year
- the next 9 dog years give 1 more human year
- every 5 next dog years give 1 extra human year

File: src/getHumanAge.test.js (hunk 1 above)
Comment (MarinaT-N) [reply src/getHumanAge.test.js]: I made a mistake in the title, I wrote " Solutin". sorry for that.

**Review by Oleksandra-Pasenchenko — APPROVED** [review]
Great job!
